x/ecocredit/client/testsuite/grpc.go

```diff
@@ -185,8 +184,7 @@ func (s *IntegrationTestSuite) TestCreditTypes() {
 	var res core.QueryCreditTypesResponse
 	err = val.ClientCtx.Codec.UnmarshalJSON(resp, &res)
 	require.NoError(err)
-	require.Equal(creditTypes, res.CreditTypes)
-
+	require.Greater(len(res.CreditTypes), 0)
```

Comment:
Hmm... I think testing equal is more ideal but it looks like this might be an issue with tracking the number of credit types added during the integration tests. Ok to leave as is though given #1041.